### DISCUSSION TITLE
Removed game still appears on the games list

### DISCUSSION THREAD

**Discussion by kathyrollo** [amakvana/YuzuModDownloader#48] [Q&A] [answered]
Hi @amakvana 

I have uninstalled all updates/DLCs from Yuzu and also deleted the `.nsp` file for Super Mario Odyssey. However, when I download mods, it's still showing up in the games list:

![Screenshot 2024-03-16 231053](https://github.com/amakvana/SwitchEmuModDownloader/assets/28589393/07d457d4-665c-4317-8acb-4c4b6d85e85f)

Wanted to ask this as a question first before raising as an issue. Is there something I need to do so it no longer downloads mods for an uninstalled/deleted game?

**Comment by amakvana** [comment, marked as answer]
Hey! 

Thanks for opening up the Discussion.

When you import your games library into Yuzu, not only does it display the games within the GUI, but it also creates the folders within `/load/` and `/cache/game_list/`. These are the two folders SwitchEmuModDownloader scans to identify which games to check against the selected repo database.

When a game is removed from Yuzu, these folders are not deleted automatically -  I assume it's to preserve the mods / settings for your game in-case you reimport it, to which the settings and mods are also reinstated. However, as these folders are not automatically deleted, they are still scanned when SwitchEmuModDownloader is executed and _this_ is what you're experiencing. 

The ideal scenario would have been the previous devs to add some sort of "delete" option next to each game, and when that's selected, a post-cleanup is performed and the corresponding folders/files are removed. 

For now, within your `/load/` and `/cache/game_list/` folders, delete any files/folders containing `0100000000010000`. This is the Game ID paired to Super Mario Odyssey.

Hope that makes sense!

Any queries, let me know.

Cheers!

**Reply by kathyrollo** [reply]
> The ideal scenario would have been the previous devs to add some sort of "delete" option next to each game, and when that's selected, a post-cleanup is performed and the corresponding folders/files are removed.

Thank you for the detailed explanation. This makes a lot of sense and would be a solid suggestion to whomever is carrying over the development of the emulator.

> For now, within your /load/ and /cache/game_list/ folders, delete any files/folders containing 0100000000010000. This is the Game ID paired to Super Mario Odyssey.

I confirm this solved the problem. In fact, I deleted all contents of both folders for a fresh start. Cheers!

**Comment by amakvana** [comment]
Glad to hear this resolved your issue! I'll close this now. 

Thanks again!
